src/packages/price/price.tsx

```
@@ -86,14 +106,15 @@ export const Price: FunctionComponent<Partial<WebPriceProps>> = (props) => {
             [`${classPrefix}-rtl`]: rtl,
           },
         ])}
+        style={priceColorStyle}
         dangerouslySetInnerHTML={{ __html: symbol || '' }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)